dist/api_data.py

```diff
@@ -630,6 +630,9 @@ def __ge__(self, other):
             seconds to wait between each checkpoint; setting this value above 0 configures
             periodic checkpoints''',
             min='0', max='100000'),
+        Config('precise_checkpoint', 'false', r'''
+            Only write stable data to the checkpoint.''',
```

Comment:
Some better documentation here would be nice - e.g. the negative effects on the rest of the system and what exactly stable means.

Reply:
done